docs/references/files/fossa-yml.md

```diff
@@ -303,6 +303,33 @@ Found target: yarn@prod/docker
 Found target: pipenv@prod
 ```
 
+### Filtering by Maven Dependency Scope
+
+You can provide maven dependency scopes that you would like to filter. You can filter by either inclusion or exclusion.
+
+#### scope-only:
+
+the list of `only` scopes that should by scanned
+
+#### scope-exclude:
+
+The list of `exclude` scopes that you would like to exclude from scanning
+
+
+```yaml
+version: 3
+
+maven:
```

Comment:
[Required] You should update spec file `fossa-yml.v3.schema.json`. You can use the schema validator online to confirm your schema works for changes. I use: https://www.jsonschemavalidator.net/ 

These files are used by various IDEs (and plugins) to provide auto complete, when user is editing .fossa-* files

Reply:
good catch thank you for that!